### PR TITLE
Formalize Docs

### PR DESCRIPTION
**Pull request by Stonepia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
-# Intel® XPU Backend for Triton\*
+# XPU Backend for Triton\*
 
-This is the development repository of Intel® XPU Backend for Triton\*, a new [OpenAI Triton](https://github.com/openai/triton) backend for Intel® GPUs. Triton is a language and compiler for writing highly efficient custom Deep-Learning primitives. The aim of Triton\* is to provide an open-source environment to write fast code at higher productivity than CUDA, but also with higher flexibility than other existing DSLs. Intel® XPU Backend for Triton\* is a module used by Triton to provide a reasonable tradeoff between performance and productivity on Intel® GPUs.
+This is the development repository of XPU Backend for Triton\*, a new [OpenAI Triton](https://github.com/openai/triton) backend for Intel GPUs. Triton is a language and compiler for writing highly efficient custom deep learning primitives. The aim of Triton is to provide an open-source environment to write fast code at higher productivity than CUDA, but also with higher flexibility than other existing DSLs. XPU Backend for Triton\* is a module used by Triton to provide a reasonable tradeoff between performance and productivity on Intel GPUs.
```
Comment (riverliuintel) [inline README.md:3]: Intel® XPU to Intel XPU

File: README.md [hunk 2]
```diff
@@ -1,44 +1,44 @@
-# Intel® XPU Backend for Triton\*
```
Comment (riverliuintel) [inline README.md:1]: Just keep Intel® XPU Backend for Triton\*, until got further clarification.

File: docs/test_docs/end_to_end_tests.md [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
     - [Debugging Tips](#debugging-tips)
   - [Profiling](#profiling)
-    - [E2E Tests Setting:](#e2e-tests-setting)
+    - [End-to-end Tests Setting:](#end-to-end-tests-setting)
       - [Profiling Settings](#profiling-settings)
       - [Profiling Tips](#profiling-tips)
 
 
 # Overview
-This doc contains [Torchdynamo Benchmarks](https://github.com/pytorch/pytorch/tree/main/benchmarks/dynamo) setup for Intel® XPU Backend for Triton\*.
```
Comment (riverliuintel) [inline docs/test_docs/end_to_end_tests.md:14]: Intel® XPU -> Intel XPU

**Review by riverliuintel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Stonepia) [reply README.md:1]: Thanks for the comment, this is done by https://github.com/intel/intel-xpu-backend-for-triton/pull/71/commits/039f935289dd5af26adaf1e7b4b5393ceb9c696f

File: README.md (hunk 1 above)
Comment (Stonepia) [reply README.md]: Thanks for the comment! This `Intel®` is for `Intel® XPU Backend for Triton*`. Thus I keep the `®` here.

File: docs/test_docs/end_to_end_tests.md (hunk 3 above)
Comment (Stonepia) [reply docs/test_docs/end_to_end_tests.md:14]: This `®` is also for `Intel® XPU Backend for Triton*`

**Review by riverliuintel — APPROVED** [review]
Looks good.

**Pull request merged**